### DISCUSSION TITLE
Amharic: What do I need to do to create am_core_web_sm

### DISCUSSION THREAD

**Discussion by yosiasz** [explosion/spaCy#6921] [Language Support]
Greetings,

Where can I go to read on how to create am_core_web_sm for Amharic?

Thanks!

**Comment by svlandeg** [comment]
Hi! This thread got buried a little bit so I pinned it again: https://github.com/explosion/spaCy/discussions/3056

You can keep the discussion here in this Amharic-specific thread, but the one I linked contains a lot of pointers and explanations on how the process usually works. Hopefully that's a good starting point for you :-)
